### PR TITLE
Testrail: add priority

### PR DESCRIPTION
**Pull request by andrey-brovko**

### CODE REVIEW

File: GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace GherkinSyncTool.Synchronizers.TestRail.Model
+{
+    public enum PriorityId : ulong
```
Comment (vladimir-s-quantori) [inline GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs:3]: Isn't `ulong` a little overkill?

**Review by SergeyRuzin — APPROVED** [review]

File: GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs (hunk 1 above)
Comment (andrey-brovko) [reply GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs]: fixed.

File: GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿namespace GherkinSyncTool.Synchronizers.TestRail.Model
 {
-    public enum PriorityId : ulong
+    public enum PriorityId : byte
```
Comment (vladimir-s-quantori) [inline GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs:3]: We can leave it as default type, without explicit statement - no need to use `byte` here

**Review by vladimir-s-quantori — APPROVED** [review]

File: GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs (hunk 2 above)
Comment (andrey-brovko) [reply GherkinSyncTool.Synchronizers.TestRail/Model/PriorityId.cs]: Fixed

**Pull request merged**